src/control/TicketService.java

```diff
@@ -47,9 +42,40 @@ public static void main(String[] args)  {
     ticket4.share(phoneNumber, email);
 
     System.out.println(ticket3.getId());
-        System.out.println(ticket4.hashCode());
+    System.out.println(ticket4.hashCode());
+
+    // Homework #5: Custom ArrayList & Custom HashSet
+
+    CustomArrayList<Integer> numbers = new CustomArrayList<>(4);
+
+    numbers.put(1);
+    numbers.put(2);
+    numbers.put(3);
+    numbers.put(4);
+    numbers.put(5);
+    System.out.println(numbers);
+
+
+    CustomHashSet<String> strings = new CustomHashSet<>(2);
```

Comment:
name collection variables as `stringsSet` for example

Reply:
Fixed. I renamed variables' names